### PR TITLE
update save-and-restore tools

### PR DESCRIPTION
**Pull request by ttkorhonen**
Updates to the (badly) outdated Save-and-Restore tools page.

### CODE REVIEW

**Comment by ttkorhonen** [comment]
Intersphinx links need adjustment...have to be fixed before merging.

**Comment by ttkorhonen** [comment]
After a few silly tries, this finally should work now.

File: getting-started/save-restore-tools.md [hunk 1]
```diff
@@ -18,11 +19,16 @@ The XAL collection of high-level applications includes 'score': A Table of PV na
...
+### CS-Studio Phoebus [Save-And-Restore](inv:phoebus:std#app/save-and-restore/app/doc/index)
 
-Similar to 'score', but simpler and still unter development. Saves values to XML files, no connection to RDB.
+The save-and-restore application can be used to take “snapshots” of a pre-defined list if PVs at a certain point in time, and write the persisted values back at some later point.
 
-For more on CSS (Control System Studio), see the [main page at DESY](http://css.desy.de).
+### CS-Studio Phoebus [PV Table](inv:phoebus:std#app/pvtable/doc/index)
+
+The PV Table provides a tabular view of PV names and their current value with time stamp and alarm state.
+You can take a “snapshotData” of current values and dates, and the table will now highlight rows where the current value differs from the snapshotData.
```
Comment (minijackson) [inline getting-started/save-restore-tools.md:29]: Also nitpick:

```suggestion
You can take a `snapshotData` of current values and dates,
and the table will now highlight rows where the current value differs from the `snapshotData`.
```

**Review by minijackson — COMMENTED** [review]
Can you squash some of your commits? The way things are right now, some commits will reference some issues, which might be confusing.

I think we also should rename the main title. I personally have no idea what "PV Save and Restore Tools available" means.

**Comment by minijackson** [comment]
I see you added some other projects in the intersphinx configuration. Are they used if we don't have any links to them? I'm not saying we should remove them, I'm just asking out of curiosity.

**Comment by ttkorhonen** [comment]
Thank you for the comments. I think I will create a new PR, this one is indeed confusing (I should not work from sofa late in the evening.)
Concerning the intersphinx links, I am planning to use them even if there are no links as of now. As far as I understand ,they are used in the build process only if the parser finds links and the overhead is low.

**Comment by ttkorhonen** [comment]
Just a final comment: this page is so much outdated that it is embarrassing, thus I wanted to update it right away. It is another question if we need the page in the first place, it is one of the doubly ported pages (ANL site -> epics-controls -> docs.epics-controls) and maybe the content would be better to have somewhere else.

**Comment by minijackson** [comment]
Since saving and restoring PVs is so common, I do think it's useful to have a page comparing the various solutions. Maybe page written from scratch would be good? Not completely sure.

**Comment by ttkorhonen** [comment]
I try to think of writing a new page but for now I would like to simply update... I just cannot look at it in the current state.

File: getting-started/save-restore-tools.md [hunk 2]
```diff
@@ -18,11 +19,16 @@ The XAL collection of high-level applications includes 'score': A Table of PV na
 
 Don't know a good web page, but try [this one](https://openxal.github.io/) .
 
-### CSS 'PV Table'
+### CS-Studio Phoebus [Save-And-Restore](inv:phoebus:std#app/save-and-restore/app/doc/index)
 
-Similar to 'score', but simpler and still unter development. Saves values to XML files, no connection to RDB.
+The save-and-restore application can be used to take “snapshots” of a pre-defined list if PVs at a certain point in time, and write the persisted values back at some later point.
```
[the remark this answers is not in the document]
Comment (ttkorhonen) [reply getting-started/save-restore-tools.md:24]: This is about changing the quote type, right? I never could figure out the different types of single and double quotes and what they are supposed to mean. My keyboard has only one variant of double quotes anyway ;-)

**Pull request closed without merging**